Loops/Loops Practice/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Practice JavaScript Loops</title>
+    <link href="css/style.css" rel="stylesheet">
+</head>
+<body>
+<main></main>
+<script src="js/app.js"></script>
+</body>
+</html>
```

Comment:
I would start adding new lines at the end of file.

Reply:
New commit pushed addressing this and directory comment above.